src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -1521,20 +1522,46 @@ public SessionResultsBundle getSessionResultsForUser(
             allQuestionsMap.put(qn.getId(), qn);
         }
 
+        List<FeedbackQuestionAttributes> questionsWithDefaultSections = new ArrayList<>();
+        if (Const.DEFAULT_SECTION.equals(section)) {
```

Comment:
When I write the refactored version, I have considered the case for `DEFAULT_SECTION`.  To me, it looks like adding support for `No specific section` does not need to modify the backend. What happened if we do not modify the logic?

Actually, `DEFAULT_SECTION` does not mean it comes from Instructors. V6 actually allows default section for students if section is not provided (P.S. the behavior need to be changed in V6).

Reply:
Looks like you're right. For some reason I didn't manage to get it work before.